dask_searchcv/methods.py

```diff
@@ -343,8 +362,11 @@ def create_cv_results(scores, candidate_params, n_splits, error_score, weights):
     return results
 
 
-def get_best_params(candidate_params, cv_results):
-    best_index = np.flatnonzero(cv_results["rank_test_score"] == 1)[0]
+def get_best_params(candidate_params, cv_results, scorer):
+    if scorer is True:
+        scorer = 'score'
```

Comment:
This took me a bit to figure out, would prefer if this logic was moved out of the function, and `'score'` was passed in the call instead.